### ISSUE TITLE
Wrong validation on keyword arguments

### ISSUE THREAD

**Issue by HelioGuilherme66** [robotframework/RIDE#1978]
Validation of keyword arguments is not respecting Robot Framework documentation:
[Named Only Arguments](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#named-only-arguments-with-user-keywords)
The below error should not occur:
![image](https://user-images.githubusercontent.com/2866630/63019518-755cd980-be93-11e9-92da-64e65d2af100.png)

The workaround is to use the Text Editor because uses RF validation rules.

```
Found Robot Framework version 3.1.2
RIDE 1.7.4b1dev1 using python version 3.7.4 with wx version 4.0.6 in win32.
```

See also #1523

**Comment by HelioGuilherme66** [comment]
Screenshot of example taken from User Guide, runs OK and validates OK on Text Editor. (Windows 7, Python 2.7.13/3.7.1, wxPython 4.0.7post2)
![image](https://user-images.githubusercontent.com/2866630/71449771-13b75080-274c-11ea-9c7e-8ef4ea632406.png)

Source test case:
[test_varargs.robot.txt](https://github.com/robotframework/RIDE/files/4000766/test_varargs.robot.txt)

**Comment by HelioGuilherme66** [comment]
Issue still present on RIDE v2.0.2.dev1 in Fedora Linux 36, Python 3.10.10, wxPython 4.0.7 (KDE Plasma 5 with darkmode, custom colors).

**Comment by HelioGuilherme66** [comment]
Fixed in 2.0.8dev3.
